opal/browser/audio/node.rb

```diff
@@ -1,42 +1,57 @@
-module Browser; module Audio; module Node;
+module Browser; module Audio; module Node
 
-module Pluggable
-  def connect(other_node)
-    `#@native.connect(#{Native.convert(other_node)})`
+class Base
+  include Native
+
+  def initialize(native)
+    @native = native
   end
 
-  def disconnect(other_node, options = {})
-    other_node = Native.try_convert(other_node)
-    output     = options[:output] || 0
-    input      = options[:input]  || 0
+  def method_missing(name, value = nil)
+    if name.end_with? '='
+      `#@native[#{name.delete '='}].value = value`
+    elsif value.nil? || value == true
+      `#@native[#{name}].value`
+    elsif value == false
+      `#@native[#{name}]`
+    else
+      super
+    end
+  end
+
+  def respond_to_missing?(method, include_all = false)
+    method_name = method.delete('=')
+    `#@native[#{method_name}] !== undefined` || super
```

Comment:
You don't have to `super` in this case. Also no need to define `method_name`, just use it directly. Also it's probably a better idea to `!= null` instead of `!== undefined`.